package.json

```diff
@@ -50,7 +49,7 @@
     "style-loader": "^0.19.1",
     "zoapp-common": "0.1.2",
     "zoapp-materialcomponents": "^0.2.1",
-    "zoapp-ui": "0.1.5"
+    "zoapp-ui": "0.2.0"
```

Comment:
This might introduce a regression related to the dialogs/dialog manager

Reply:
how ?

Reply:
I think there are still files where we import `DialogManager` from `zoapp-ui`:

```
import { DialogManager } from "zoapp-ui";
```

and I dont think we have it anymore in `zoapp-ui:0.2.0`

Reply:
We don't have to use DialogManager, but I will export it ok.

Reply:
Yes I know but by keeping `zoapp-ui:0.1.5` we can progressively migrate. Here that breaks everything until we use `Rmdc.showDialog()` everywhere :D

Reply:
I will fully migrate Opla/front to Rmdc.showDialog() tomorrow ;-) And Opla/front rely on zoapp-front 0.1.2. I will update it to 0.2.0 today.

Reply:
I exported DialogManager from zoapp-materialcomponents for emergency cases. Like your displayWebServiceEditor ;-)